app/controllers/call_lists_controller.rb

```diff
@@ -18,21 +20,20 @@ def remove_patient
   end
 
   def clear_current_user_call_list
-    current_user.clear_call_list
+    current_user.clear_call_list current_line
     respond_to do |format|
       format.js { render template: 'users/refresh_patients', layout: false }
     end
   end
 
   def reorder_call_list
-    current_user.reorder_call_list params[:order]
+    current_user.reorder_call_list params[:order], current_line
     head :ok
   end
 
   private
 
   def retrieve_patients
     @patient = Patient.find params[:id]
-    @urgent_patient = Patient.where(urgent_flag: true)
```

Comment:
where'd this one go? :eyes:

Reply:
believe it or not, unnecessary db call! Saves us some overhead.